simplified-app-shared/src/main/res/layout/login_dialog.xml

```diff
@@ -76,7 +76,6 @@
         android:layout_weight="1"
         android:inputType="textNoSuggestions"
         android:minWidth="160dp"
-        android:maxLength="4"
```

Comment:
There are new keys in Accounts.json for a library that dictate whether or not a library should be allowing alphanumeric passwords as well as the maximum allowed length (with 0 being unlimited). Refer to the iOS code [here](https://github.com/NYPL-Simplified/Simplified-iOS/blob/340df4266f82fc9ba643034a334db467c2aa4dd0/Simplified/NYPLSettingsAccountDetailViewController.m#L171) and [here](https://github.com/NYPL-Simplified/Simplified-iOS/blob/340df4266f82fc9ba643034a334db467c2aa4dd0/Simplified/NYPLSettingsAccountDetailViewController.m#L1275) and let me know if you need assistance implementing.